akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/PresentationPods.scala

```diff
@@ -176,8 +176,8 @@ case class PresentationPod(id: String, currentPresenter: String,
     // 100D-checkedWidth is the maximum the page can be moved over
     val checkedWidth = Math.min(widthRatio, 100D) //if (widthRatio <= 100D) widthRatio else 100D
     val checkedHeight = Math.min(heightRatio, 100D)
-    val checkedXOffset = Math.min(xOffset, 0D)
-    val checkedYOffset = Math.min(yOffset, 0D)
+    val checkedXOffset = xOffset
```

Comment:
@KDSBrowne why  remove the flooring insurance on these two?

Reply:
It was interfering with `akka`'s ability to send negative values to the client. These negative values were needed for the viewers, as they enable the viewer's to follow the presenter when they pan across the infinite canvas.